packages/palette/src/elements/Label/Label.tsx

```diff
@@ -1,4 +1,4 @@
-import { Text } from "@artsy/palette"
+import { Text } from "../Text"
```

Comment:
😬

Reply:
this was wrong, right? importing from @artsy/palette within palette?!

Reply:
You are indeed correct. Surprised this was working at all.